salt/states/ssh_known_hosts.py

```diff
@@ -12,6 +12,7 @@
         - present
         - user: root
         - fingerprint: 16:27:ac:a5:76:28:2d:36:63:1b:56:4d:eb:df:a6:48
+        - fingerprint_hash_type: md5
```

Comment:
@nixjdm Did you want to change the examples to SHA-256 and away from MD5? If not, we can have somebody else do it and then come back to this PR after that is done.